### PR TITLE
feat: add netdata-restarter cronjob

### PR DESCRIPTION
**Pull request by ilyam8**
This PR adds netdata restarter cronjob. Implements the 2nd item from #337.

Tested on local k8s cluster.

### CODE REVIEW

File: charts/netdata/templates/cronjob.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{{- if include "netdata.restarter.enabled" . }}
+---
+{{- $cmdList := list }}
+{{- $cmd := printf "kubectl rollout restart deployment %s-parent" (include "netdata.name" .) }}
+{{- $cmdList = append $cmdList (ternary $cmd "" .Values.parent.enabled) }}
+{{- $cmd = printf "kubectl rollout restart daemonset %s-child" (include "netdata.name" .) }}
+{{- $cmdList = append $cmdList (ternary $cmd "" .Values.child.enabled) }}
+{{- $cmd = printf "kubectl rollout restart deployment %s-k8s-state" (include "netdata.name" .) }}
+{{- $cmdList = append $cmdList (ternary $cmd "" .Values.k8sState.enabled) }}
+{{- $cmdList = compact $cmdList }}
```
Comment (ilyam8) [inline charts/netdata/templates/cronjob.yaml:10]: I couldn't find a way to make it less verbose. If you have any ideas, please suggest them.

File: charts/netdata/templates/cronjob.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: {{ .Values.serviceAccount.name }}-restarter
+          restartPolicy: {{ .Values.restarter.restartPolicy }}
+          containers:
+            - name: netdata-restarter
+              image: {{ .Values.restarter.image.repository }}:{{ .Values.restarter.image.tag }}
+              imagePullPolicy: {{ .Values.restarter.image.pullPolicy }}
```
Comment (witalisoft) [inline charts/netdata/templates/cronjob.yaml:30]: nit: good to consider `imagePullSecrets`

File: charts/netdata/templates/_helpers.tpl [hunk 3]
```diff
@@ -52,3 +52,14 @@ Return the appropriate apiVersion for ingress.
 {{- "networking.k8s.io/v1" -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Return a value indicating whether the restarter is enabled.
+*/}}
+{{- define "netdata.restarter.enabled" -}}
+{{- if and .Values.restarter.enabled (or .Values.parent.enabled .Values.child.enabled .Values.k8sState.enabled) }}
```
Comment (witalisoft) [inline charts/netdata/templates/_helpers.tpl:60]: netdata restarter has only practical sense `image.pullPolicy` to `Always`

**Review by witalisoft — DISMISSED** [review]

**Review by M4itee — DISMISSED** [review]
I would just mention in the docs that the purpose of this restarter to be there in a first place is fire and forget update process. But this is not the blocker for it!

**Review by M4itee — APPROVED** [review]

**Review by witalisoft — APPROVED** [review]

**Pull request merged**